react-frontend/src/components/StyleComponents/card.js

```diff
@@ -154,14 +154,18 @@ export const CommunityCardStyled = styled(Link).attrs({
   className: 'cardRound',
 })`
   background: #003366;
-  border-radius: ${cardBorderRadius};
+  border-radius: ${cardBorderRadius} ${cardBorderRadius} 30px 30px;
   height: 95%;
   margin-bottom: 20px;
   width: 100%;
   display: flex;
   flex-direction: column;
   :hover {
```

Comment:
Could you please change this to `:hover, :focus` (or whatever proper syntax is) ... so that the properties below apply to the cards in both hover and focus states? See below.